### DISCUSSION TITLE
compiler can't find @chainlink contract

### DISCUSSION THREAD

**Discussion by muhyomoto** [smartcontractkit/full-blockchain-solidity-course-py#1541] [Q&A] [answered]
heres the error msg.  file is contracts\test\VRFCoordinatorV2Mock.sol
thanks for any ideas to fix

CompilerError: solc returned the following errors:

ParserError: Source "C:/Users/crypt/.brownie/packages/smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.8/interfaces/LinkTokenInterface.sol" not found: File not found.
 --> contracts/test/VRFCoordinatorV2Mock.sol:5:1:
  |
5 | import "@chainlink/contracts/src/v0.8/interfaces/LinkTokenInterface.sol";

**Comment by muhyomoto** [comment]
heres all the  brownie compile msg

PS C:\Users\crypt\smartcontract-lottery> brownie compile
INFO: Could not find files for the given pattern(s).
Brownie v1.18.1 - Python development framework for Ethereum

New compatible solc version available: 0.8.9
Compiling contracts...
  Solc version: 0.6.12
  Optimizer: Enabled  Runs: 200
  EVM Version: Istanbul
Generating build data...
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/LinkTokenReceiver
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/ChainlinkRequestInterface
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/LinkTokenInterface
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/SafeMathChainlink
 - MockOracle

Compiling contracts...
  Solc version: 0.4.26
  Optimizer: Enabled  Runs: 200
  EVM Version: Byzantium
Generating build data...
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/ERC677Token
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/ERC20
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/ERC20Basic
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/ERC677
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/ERC677Receiver
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/BasicToken
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/SafeMathChainlink
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/StandardToken
 - LinkToken

Compiling contracts...
  Solc version: 0.8.9
  Optimizer: Enabled  Runs: 200
  EVM Version: Istanbul
CompilerError: solc returned the following errors:

ParserError: Source "C:/Users/crypt/.brownie/packages/smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.8/interfaces/LinkTokenInterface.sol" not found: File not found.
 --> contracts/test/VRFCoordinatorV2Mock.sol:5:1:
  |
5 | import "@chainlink/contracts/src/v0.8/interfaces/LinkTokenInterface.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^


ParserError: Source "C:/Users/crypt/.brownie/packages/smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.8/interfaces/VRFCoordinatorV2Interface.sol" not found: File not found.
 --> contracts/test/VRFCoordinatorV2Mock.sol:6:1:
  |
6 | import "@chainlink/contracts/src/v0.8/interfaces/VRFCoordinatorV2Interface.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^


ParserError: Source "C:/Users/crypt/.brownie/packages/smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.8/VRFConsumerBaseV2.sol" not found: File not found.
 --> contracts/test/VRFCoordinatorV2Mock.sol:7:1:
  |
7 | import "@chainlink/contracts/src/v0.8/VRFConsumerBaseV2.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

PS C:\Users\crypt\smartcontract-lottery>

**Comment by muhyomoto** [comment]
whoops!  wrong file.  should be VRFCoordinatorMock.sol  which compiles fine

**Comment by cromewar** [comment, marked as answer]
Hello @muhyomoto 
I'm glad you problem is solved now, closing this.
